index.js

```diff
@@ -119,3 +119,22 @@ const portfolioCard = (pro) => {
   btndiv.append(btn);
 };
 portfolio.forEach((pro) => portfolioCard(pro));
+
+function validate() {
+  const contactForm = document.getElementById('contactForm');
+  const email = document.getElementById('email').value;
+  const text = document.getElementById('text');
+  const pattern = /^[a-z]\w+@[a-z_]+?\.[a-z]{2,3}$/;
```

Comment:
- [ ] For this project, users should use only emails with lowercase characters. for now, our website validation allows emails with uppercase characters to be submitted. Please, kindly fix this issue so that only emails with lowercase characters can be submitted. you can do that by using the JS method `toLowerCase()`. for example  : `email === email.toLowerCase(); `